pkgs/servers/mastodon/source.nix

```diff
@@ -2,8 +2,8 @@
 { fetchgit, applyPatches }: let
   src = fetchgit {
```

Comment:
nit: this should use fetchFromGitHub which should download a archive which is faster and more reproducible.